package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1119,10 +1128,9 @@ def principal_axes(group, pbc=None):
         .. versionchanged:: 0.8 Added *pbc* keyword
 
         """
-        atomgroup = group.atoms
```

Comment:
@richardjgowers @jbarnoud Is there a reason for using `atomgroup = group.atoms `
in place of directly using `group.moment_of_inertia`?

Reply:
Here it should be fine since MOI is doing the conversion.  In general these methods are attached to **all** groups, so `ResidueGroup` too, but they want to operate on atoms.  So for a RG, `atomgroup = group.atoms` is converting to an AtomGroup within the method.